src/main/java/seedu/address/ui/CommandBox.java

```diff
@@ -69,6 +73,19 @@ private void setStyleToIndicateCommandFailure() {
         styleClass.add(ERROR_STYLE_CLASS);
     }
 
+    private void onInputChange(String oldValue, String newValue) {
+        // Your logic to handle input changes here
+        if (oldValue.contains(FindCommand.COMMAND_WORD) && !newValue.isEmpty()
+            && (!newValue.trim().equals(FindCommand.COMMAND_WORD))) {
```

Comment:
Maybe next time this kind of conditions can abstract out to a helper function for better readability